pump/storage/storage.go

```diff
@@ -654,6 +663,10 @@ func (a *Append) doGCTS(ts int64) {
 			}
 			break
 		}
+
```

Comment:
add log for kv gc and vlog gc

Reply:
add in https://github.com/pingcap/tidb-binlog/pull/648/commits/5e91e874a6db935d100e92c87c540e58663a0a39